portfolio/src/main/webapp/script-comments.js

```diff
@@ -0,0 +1,277 @@
+  for (var i = 1; i <= numPages; i++) {
+    if (i == activePageNumber) {
+      pageListElement.appendChild(createPageElement(i, true));
+    } else {
+      pageListElement.appendChild(createPageElement(i, false));
+    }
+  }
+  pageListElement.appendChild(createPageElement('Next'));
+
+  return activePageNumber;
+}
+
+/**
+ * Creates an <li> element for a page item in the pagination bar. 
+ * @param {string} text The innerText of the <a> child element of the <li>.
+ * @param {boolean} active Whether the created page item is active.
+ * @return {!Element<li>} The page element created.
+ */
+function createPageElement(text, active) {
+  const pageElement = document.createElement('li');
+  pageElement.className = 'page-item';
+  if (active) { 
+    pageElement.classList.add('active'); 
+  }
+  
+  const linkElement = document.createElement('a');
+  linkElement.className = 'page-link';
+  linkElement.innerText = text;
+  pageElement.appendChild(linkElement);
+
+  pageElement.addEventListener('click', event => changePage(event));
+  return pageElement;
+}
+
+/**
+ * Changes the active page element and displays the comments on that page when a page element is clicked.
+ * @param {!Event} event The click event that triggered the change in active page.
+ */
+function changePage(event) {
+  const currentPage = document.querySelector('.page-item.active');
+  const pageText = event.currentTarget.firstElementChild.innerText;
+  
+  if (pageText == 'Previous') {
+    if (currentPage.firstElementChild.innerText == 1) { 
+      // Current page is the first page, no previous page to open.
+      return; 
+    }
+    currentPage.previousSibling.classList.add('active');
+  } else if (pageText == 'Next') {
+    const numPages = document.querySelectorAll('#page-list li').length;
+    if (currentPage.firstElementChild.innerText == numPages - 2) { 
+      // Current page is the last page, no next page to open.
+      return; 
+    }
+    currentPage.nextSibling.classList.add('active');
+  } else {
+    event.currentTarget.classList.add('active');
+  }
+
+  currentPage.classList.remove('active');
+
+  // Fetch comments on that page without recreating pagination in the current language.
+  fetchTranslatedComments(false);
+}
+
+/** 
+ * Fetches the comments in the langauge specified. 
+ * @param {boolean=} createNewPagination Whether a new pagination bar should be created.
+ */
+function fetchTranslatedComments(createNewPagination = false) {
+  const languageCode = document.getElementById('language').value;
+  fetchComments(createNewPagination, languageCode);
+}
+
+/**
+ * Fetches the comments from the data server and displays them on the currently active page.
+ * @param {boolean=} createNewPagination Whether a new pagination bar should be created.
+ * @param {string=} languageCode The language in which the comments will be shown.
+ */
+function fetchComments(createNewPagination = false, languageCode = 'original') {
```

Comment:
How about `fetchActivePageComments`?

You want to be specific about what your function is doing.

Reply:
got it